pkg/webhooks/utils/exclude.go

```diff
@@ -6,11 +6,15 @@ func ExcludeKyvernoResources(kind string) bool {
 		return true
 	case "PolicyReport":
 		return true
-	case "ReportChangeRequest":
+	case "AdmissionReport":
 		return true
-	case "GenerateRequest":
+	case "ClusterAdmissionReport":
+		return true
+	case "BackgroundScanReport":
 		return true
-	case "ClusterReportChangeRequest":
+	case "ClusterBackgroundScanReport":
+		return true
+	case "GenerateRequest":
```

Comment:
We have to add missing `UpdateRequest` here as well

Reply:
added